package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "app.js",
   "scripts": {
-    "test": "NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register ./server/test",
+    "test": "NODE_ENV=test nyc --reporter=html --reporter=text mocha ./server/test/  --require @babel/register --recursive",
```

Comment:
Change this script so that it runs only one index.spec.js. With this, you can define an order of tests

E.g index.spec.js
```
import "./models/users.spec";
import "./models/favourites.spec";
import "./models/comments.spec";
```
so now we can run tests in this order

Reply:
Ok